hana/enable_cost_optimized.sls

```diff
@@ -0,0 +1,58 @@
+{%- from "hana/map.jinja" import hana with context -%}
+{% set host = grains['host'] %}
+
+{% for node in hana.nodes %}
+{% if node.host == host and node.scenario_type is defined and node.scenario_type.lower() == 'cost-optimized' %}
+
+reduce_memory_resources_{{  node.host+node.sid }}:
+    hana.memory_resources_updated:
+      - name: {{  node.host }}
+      {% if node.cost_optimized_parameters is defined %}
```

Comment:
If I'm not wrong the next 2 parameters are mandatory, right? Why do you check if this entry is defined. It must be defined. We should think about a better way to do that.
The formula even must fail if they are not defined, but the formula rendering must fail, not the `memory_resources_updated` call